code/modules/client/preference_setup/general/04_equipment.dm

```diff
@@ -7,31 +7,28 @@
 	S["undershirt"]	>> pref.undershirt
 	S["socks"]		>> pref.socks
 	S["backbag"]	>> pref.backbag
-	S["gear"]		>> pref.gear
 
 /datum/category_item/player_setup_item/general/equipment/save_character(var/savefile/S)
 	S["underwear"]	<< pref.underwear
 	S["undershirt"]	<< pref.undershirt
 	S["socks"]		<< pref.socks
 	S["backbag"]	<< pref.backbag
-	S["gear"]		<< pref.gear
 
 /datum/category_item/player_setup_item/general/equipment/gather_load_query()
-	return list("ss13_characters" = list("vars" = list("underwear", "undershirt", "socks", "backbag", "gear"), "args" = list("id")))
+	return list("ss13_characters" = list("vars" = list("underwear", "undershirt", "socks", "backbag"), "args" = list("id")))
 
 /datum/category_item/player_setup_item/general/equipment/gather_load_parameters()
 	return list(":id" = pref.current_character)
 
 /datum/category_item/player_setup_item/general/equipment/gather_save_query()
-	return list("ss13_characters" = list("underwear", "undershirt", "socks", "backbag", "gear", "id" = 1, "ckey" = 1))
+	return list("ss13_characters" = list("underwear", "undershirt", "socks", "backbag", "id" = 1, "ckey" = 1))
 
 /datum/category_item/player_setup_item/general/equipment/gather_save_parameters()
-	return list(":underwear" = pref.underwear, ":undershirt" = pref.undershirt, ":socks" = pref.socks, ":backbag" = pref.backbag, ":gear" = list2params(pref.gear), ":id" = pref.current_character, ":ckey" = pref.client.ckey)
+	return list(":underwear" = pref.underwear, ":undershirt" = pref.undershirt, ":socks" = pref.socks, ":backbag" = pref.backbag, ":id" = pref.current_character, ":ckey" = pref.client.ckey)
 
 /datum/category_item/player_setup_item/general/equipment/sanitize_character(var/sql_load = 0)
 	if (sql_load)
 		pref.backbag = text2num(pref.backbag)
-		pref.gear	= params2list(pref.gear)
 
 	pref.backbag	= sanitize_integer(pref.backbag, 1, backbaglist.len, initial(pref.backbag))
 

```

Comment:
Line 35 & 36 should also be removed. They're handled in the other .dm file.

this thing:
```
    if (!islist(pref.gear))
        pref.gear = list()
```